types/node/net.d.ts

```diff
@@ -736,6 +736,11 @@ declare module "net" {
          */
         check(address: SocketAddress): boolean;
         check(address: string, type?: IPVersion): boolean;
+        /**
+         * The list of rules added to the blocklist.
+         * @since v15.0.0, v14.18.0
+         */
+        rules: string[];
```

Comment:
It is a getter - https://github.com/nodejs/node/blob/e1d3ad422421cb1f1c94eb4fd9355df72c10808e/lib/internal/blocklist.js
So it's better to change to `readonly rules: string[];`